generator/sbpg/test_structs.py

```diff
@@ -99,7 +99,8 @@ def __init__(self, raw_packet, msg_type, raw_json, msg, sbp, test_msg_data=None)
     self.payload = base64.standard_b64decode(self.sbp['payload'])
     self.payload_as_byte_array = list(bytearray(self.payload))
     self.payload_len_for_encoding_buf = len(self.payload) if len(self.payload) > 0 else 1
-    msg_name = msg['name']
+    if msg_name is None:
+        msg_name = msg['name']
```

Comment:
I think this is something that got broken in json2test. this should only be used in the `from_msg` function and otherwise have the same behavior as before